viewer/packages/tools/src/HtmlOverlay/HtmlOverlayTool.test.ts

```diff
@@ -81,6 +81,7 @@ describe('HtmlOverlayTool', () => {
 
     // Act
     helper.add(htmlElement, position);
+    helper.forceUpdate();
```

Comment:
Currently `helper.add` will only _schedule_ an update because it doesn't make sense to update on each add when adding many elements. Hence, we need to forceUpdate in tests to ensure DOM is up to date.